### PR TITLE
refactor: audit package using jest

### PR DESCRIPTION
**Pull request by juanpicado**
First try for https://github.com/pnpm/pnpm/issues/2858

- audit package

### CODE REVIEW

File: packages/audit/package.json [hunk 1]
```diff
@@ -13,8 +13,7 @@
   },
   "scripts": {
     "lint": "eslint -c ../../eslint.json src/**/*.ts test/**/*.ts",
-    "_test": "cd ../.. && c8 --reporter lcov --reports-dir packages/audit/coverage ts-node packages/audit/test --type-check",
-    "test": "pnpm run compile && pnpm run _test",
+    "test": "pnpx jest",
```
Comment (zkochan) [inline packages/audit/package.json]: you can just use jest here. `pnpm run` adds the root `node_modules/.bin` to the PATH
```suggestion
    "test": "jest",
```

**Comment by juanpicado** [comment]
Travis is failing because runs `_test`. 🤔  ... Am I missing something?

File: packages/audit/package.json [hunk 2]
```diff
@@ -13,8 +13,7 @@
   },
   "scripts": {
     "lint": "eslint -c ../../eslint.json src/**/*.ts test/**/*.ts",
-    "_test": "cd ../.. && c8 --reporter lcov --reports-dir packages/audit/coverage ts-node packages/audit/test --type-check",
-    "test": "pnpm run compile && pnpm run _test",
+    "test": "jest",
```
Comment (zkochan) [inline packages/audit/package.json]: maybe try:

```suggestion
    "_test": "jest",
    "test": "pnpm run compile && pnpm run _test",
```

also, is coverage calculated by default? we need coverage

File: packages/audit/package.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
   },
   "scripts": {
     "lint": "eslint -c ../../eslint.json src/**/*.ts test/**/*.ts",
-    "_test": "jest",
+    "_test": "jest --coverage",
```
Comment (juanpicado) [inline packages/audit/package.json]: We can enable it via config for all packages. Like this https://github.com/verdaccio/verdaccio/blob/5.x/jest/config.js#L7 and ignore certain folders that won't be covered.

Comment (zkochan) [reply packages/audit/package.json]: ok

Comment (zkochan) [reply packages/audit/package.json]: I'll push that

**Comment by zkochan** [comment]
travis is dead

**Comment by juanpicado** [comment]
Yeah 😢 , it is freeze.  Maybe a different debate, why not GitHub Actions?

**Comment by zkochan** [comment]
It is mostly working OK.

**Review by zkochan — APPROVED** [review]

**Comment by juanpicado** [comment]
🎉 my first PR at pnpm 🕺

**Pull request merged**